### PR TITLE
Dynamically handle request to single component reload #369

### PR DESCRIPTION
**Pull request by pmi**

### CODE REVIEW

File: src/pages/[[...contentPath]].tsx [hunk 1]
```diff
@@ -22,22 +22,9 @@ const query = `query($path: ID) {
...
-}
-
 export async function getStaticProps(context: Context) {
-    const path = context.previewData?.contentPath || context.params?.contentPath || [];
-
-    if (context.preview) {
-        populateXPHeaders(context);
-    }
+    const path = context.params?.contentPath || [];
+    console.info(`static page ${context.preview ? '(preview) ' : ''}at: ${path}`);
```
Comment (poi33) [inline src/pages/[[...contentPath]].tsx:27]: Wont this print for every static page request?

File: src/pages/_component.tsx [hunk 2]
```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import {Context, fetchContent} from "../_enonicAdapter/guillotine/fetchContent";
+import MainView from "../_enonicAdapter/views/MainView";
+import {IS_DEV_MODE, RENDER_MODE} from '../_enonicAdapter/utils';
+
+//TODO: refactor this one to ONLY handle component requests
+export async function getServerSideProps(context: Context) {
+    const path = context.query.contentPath || [];
+    console.info(`dynamic component at: ${path}`);
```
Comment (poi33) [inline src/pages/_component.tsx:9]: Every time we get the server props we log it? Seem excessive

File: src/pages/api/preview.ts [hunk 3]
```diff
@@ -11,17 +9,12 @@ export default async function handler(req: any, res: any) {
...
-        headers: {
-            [FROM_XP_PARAM]: req.headers[FROM_XP_PARAM],
-            [RENDER_MODE_HEADER]: req.headers[RENDER_MODE_HEADER],
-            [COMPONENT_SUBPATH_HEADER]: req.headers[COMPONENT_SUBPATH_HEADER],
-            [XP_BASE_URL_HEADER]: req.headers[XP_BASE_URL_HEADER],
-        }
-    }, {
-        maxAge: 10  // sec
-    });
+    console.info(`Previewing [${path}]...`);
```
Comment (poi33) [inline src/pages/api/preview.ts:12]: Every preview is logged to oconsole?

**Review by poi33 — APPROVED** [review]
The logging is a bit excessive. 
Might be okay for the demo 🤷

File: src/pages/[[...contentPath]].tsx (hunk 1 above)
Comment (pmi) [reply src/pages/[[...contentPath]].tsx:27]: Yeah, I though that is nice to see for the demo. Looks like this in logs:
```
Previewing [/movies?mode=edit]...
static page  (preview) at: movies
static page  (preview) at: movies
static page  (preview) at:
Revalidating [/movies]...
static page  at: movies
static page  (preview) at: movies
dynamic component at: movies
Revalidating [/movies] done
```

Comment (pmi) [reply src/pages/[[...contentPath]].tsx]: I updated logging to a human language :) 
`Accessing static page  (preview) at: movies`
and
`Accessing dynamic component at: movies`

**Review by poi33 — APPROVED** [review]

**Pull request merged**